socorro/external/boto/crashstorage.py

```diff
@@ -28,13 +30,101 @@ def wait_time_generator():
         yield i
 
 
+class CrashidMissingDatestamp(Exception):
```

Comment:
There's a `CrashIDNotFound` exception and then this one `CrashidMissingDateStamp`. I should change this to `CrashIDMissingDatestamp` for consistency.